cli/src/config-schema.json

```diff
@@ -380,6 +380,12 @@
                             "type": "string"
                         }
                     },
+                    "diff-expected-exit-codes": {
+                        "type": "array",
+                        "items": {
+                            "type": "integer"
+                        }
```

Comment:
nit: can you add "description" and "default"? I think they can be copied from `merge-conflict-exit-codes`.